js/index.js

```diff
@@ -1,19 +1,23 @@
 'use strict';
 
-function Project (name, location) {
-  this.name = name;
-  this.location = location;
-  this.append = function append() {
-    //stuff happens here
-  };
-}
-
 function hamburgerClick () {
   $('.social').attr('display', 'inline')
 }
 
 $('#hamburger').on('click', hamburgerClick())
 
+function Project (name, location) {
```

Comment:
Your constructor should take an object instead of multiple arguments, just like in the lab assignment.